README.md

```diff
@@ -1,31 +1,31 @@
 # lf-sample-repository-api-dotnet-srv
 
-Sample .NET core service app that connects to a Laserfiche Cloud Repository using a service principal account.
+Sample .NET core service app that connects to a Laserfiche Cloud or Self-Hosted Repository.
 [Sample Code](./Program.cs)
 
-### Prerequisites
+## Cloud Prerequisites
 
-#### Software Prerequisites
+### Software Prerequisites
 
-- Visual Studio Code
 - .NET 6.0 core SDK
-- Git
-- Clone this repo on your local machine
+- CA, EU, or US Cloud Web Client account
 
-#### 1. Create a Service Principal
+### 1. Create a Service Principal
 
 - Log in to your account using Web Client as an administrator:
+
```

Comment:
It looks like this extra line causes extra spacing in the markdown view
![image](https://user-images.githubusercontent.com/102689343/202302733-330895d0-8b93-4a33-840d-15eb8c6f8957.png)

Reply:
ok got rid of it

Reply:
It looks like the extra whitespace is still there
https://github.com/Laserfiche/lf-sample-repository-api-dotnet-srv/blob/story/402327-add-selfhosted-documentation/README.md

Reply:
ok so after removing the other whitespace below, now the spacing matches the spacing in step 2